src/include/platform/OTARequestorInterface.h

```diff
@@ -158,9 +183,13 @@ class OTARequestorInterface
     // Destructor
     virtual ~OTARequestorInterface() = default;
 
-    // Send QueryImage command
+    // Application API to send the QueryImage command and start the image update process with the next available Provider
     virtual OTATriggerResult TriggerImmediateQuery() = 0;
 
+    // Internal API meant for use by OTARequestorDriver to send the QueryImage command and start the image update process
+    // with the Provider currently set in the OTARequestor
+    virtual void TriggerImmediateQueryInternal() = 0;
```

Comment:
Is this really necessary to have in the Interface? Is this method still relevant if someone implements something different than the SDK implementation?

Reply:
>Is this really necessary to have in the Interface? 

Yes. Because of the changes that were made in December (by @Damian-Nordic, I believe) all Driver-to-Requestor calls go through the Interface. 

> Is this method still relevant if someone implements something different than the SDK implementation?

Yes, there will still be a need to have an API to trigger a query using the Provider that had been previously set in the OTARequestor (that's what this API does)